reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,69 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][] and [Microsoft Enterprise Agreements][enterprise-agreement].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options.
+
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern].
+This support lifecycle is intended to keep customers up-to-date with the latest versions.
+
+The version 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+
+> [!IMPORTANT]
```

Comment:
Better to use `### Important` header?

Reply:
This was intentional. [!IMPORTANT] is an OPS extension. See https://review.docs.microsoft.com/en-us/help/ops-crr/openpublishing/docs/partnerdocs/gfm?toc=/help/contribute/TOC.json&bc=/help/breadcrumb/toc.json&branch=master#markdown-extensions-supported-by-open-publishing

Reply:
Got it